react-app-drawer-component/my-app/src/AppDrawer.js

```diff
@@ -0,0 +1,44 @@
+import { useState } from 'react';
+import { MdMenu, MdClear } from 'react-icons/md';
+import './AppDrawer.css';
+
+export default function AppDrawer({ heading, items }) {
+  const [showDrawer, setShowDrawer] = useState(false);
+
+  const toggleDrawer = () => setShowDrawer(!showDrawer);
+
+  const handleCloseDrawer = () => setShowDrawer(false);
+
+  const handleItemClick = (onClick) => {
+    handleCloseDrawer();
+    onClick();
```

Comment:
It took me a bit to figure out what `onClick` is, but once I did I found it pretty clever.

You should document what the `items` are in a JSDoc. I didn't realize you were passing both the menu name and an `onClick` function.